### PR TITLE
Airqo mobile setup

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)
- Redesigns of the mobile app using flutter

#### Is this change ready to hit production in its current state?
- [ ] Yes
- [x] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?

1. Install flutter on your computer
 [Windows  Install - Flutter](https://flutter.dev/docs/get-started/install/windows)
[macOs Install - Flutter](https://flutter.dev/docs/get-started/install/macos)

2. Run the app
    `cd AirQo-frontend/mobile-app`

Add [secret.dart](https://drive.google.com/file/d/1ZAjdw_phpsxBFWpz6e02tDFF4xx411V1/view?usp=sharing) to `lib/config` folder

For android, add [app.properties](https://drive.google.com/file/d/1Bktg3ckv-yO4X8Hu5QqVN7qp9LVqzWJG/view?usp=sharing) file to,  in  `AirQo-frontend/mobile-app/android` directory

For IOS, add the value [GOOGLE MAPS API Key](https://docs.google.com/document/d/1QawFn5Sfp3eOUODb38dLFsIVrU-erFpJeC7OEbWS_9Q/edit?usp=sharing),  in  `ios/Runner/AppDelegate.swift`
`GMSServices.provideAPIKey("<INSERT KEY HERE>")`

   `flutter run --no-sound-null-safety`

#### What are the relevant tickets?
- [PLAT-549](https://airqoteam.atlassian.net/browse/PLAT-549?atlOrigin=eyJpIjoiZjI2YjhmNjlhZGVhNDViNGExMWQ3YzJiMmY1MjdhZTYiLCJwIjoiaiJ9)
- [PLAT-539](https://airqoteam.atlassian.net/browse/PLAT-539?atlOrigin=eyJpIjoiNDliYWIxNzk3MjFjNGZiZTgxYjc3MGU3ZGI2YWViMWEiLCJwIjoiaiJ9)
- [PLAT-555](https://airqoteam.atlassian.net/browse/PLAT-555?atlOrigin=eyJpIjoiMTEyNWIwMWM5MjIzNDM3MDgxYTA5MDBmOTU1ZTUxMDYiLCJwIjoiaiJ9)

#### Screenshots (optional)

### CODE REVIEW

**Review by Baalmart — APPROVED** [review]
LGTM, works as expected on the IOS emulator.

**Review by danielogen — APPROVED** [review]
This is working well. I found [this](https://blog.usejournal.com/how-to-handle-api-urls-for-different-environments-in-flutter-bc4d6daf7e47) article quite useful for the separation of environments and adopting the use of `.envs`.  You can check it out.

**Comment by danielogen** [comment]
> This is working well. I found [this](https://blog.usejournal.com/how-to-handle-api-urls-for-different-environments-in-flutter-bc4d6daf7e47) article quite useful for the separation of environments and adopting the use of `.envs`. You can check it out.

cc: @NoahNsimbe

**Comment by NoahNsimbe** [comment]
> > This is working well. I found [this](https://blog.usejournal.com/how-to-handle-api-urls-for-different-environments-in-flutter-bc4d6daf7e47) article quite useful for the separation of environments and adopting the use of `.envs`. You can check it out.
> 
> cc: @NoahNsimbe

Thanks @danielogen

**Comment by danielogen** [comment]
Could we move `secret.dart` and `Google Map API Key` to google drive folder? Slack might archive the records sooner or later

File: mobile-app/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# app
+
+Airqo mobile application.
```
[the remark this answers is not in the document]
Comment (NoahNsimbe) [reply mobile-app/README.md]: done @Baalmart

**Pull request merged**